packages/commonwealth/server/CommonwealthConsumer/messageProcessors/snapshotConsumer.ts

```diff
@@ -0,0 +1,91 @@
+import axios from 'axios';
+import {factory, formatFilename} from 'common-common/src/logging';
+import {SnapshotNotification} from '../../../shared/types';
+import {DB} from "../../models";
+
+
+export async function processSnapshotMessage(this: { models: DB }, data: SnapshotNotification) {
```

Comment:
Why are we using "this" as a function argument? We should prefer a class, if we're going to be keeping this persistent models object around, at least is my sense.

Reply:
for rich/tim